public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+      "setup": "<i>1.</i> Choisissez 3 clairières de la patrie, chacune adjacente l'une à l'autre.<br/><i>2.</i> Placez 2 <strong>guerriers</strong> dans chacune de vos patries. Placez 1 <strong>guerrier</strong> dans l'autre clairière.<br/><i>3.</i> Placez le <strong>jeton de donjon</strong> dans l'une de vos patries, non adjacente à un ennemi patrie si possible. Mettez 1 <strong>scierie</strong>, <strong>atelier</strong> et <strong>recruteur</strong> sur la carte, chacun dans une patrie différente de la vôtre.<br/>< i>4.</i> Remplissez votre piste Bâtiments avec vos <strong>bâtiments</strong>, à l'exception des espaces les plus à gauche."
+    },
+    "riverfolk": {
+      "name": "Compagnie Riverfolk",
+      "summaryTitle": "Joueur la Compagnie Riverfolk",
+      "summary": "<p>En tant que Riverfolk Company, vous sillonnez les rivières serpentant à travers la grande forêt, offrant vos services à toute faction qui peut payer. Vous marquez des points de victoire en établissant des <strong>comptoirs commerciaux</strong> dans les clairières.</ p><p>Bien que la construction de comptoirs commerciaux soit un moyen viable de marquer des points, l'accumulation brute de richesse l'est également. À chaque chant d'oiseau, vous marquez des points en fonction du nombre de <strong>fonds</strong> que vous avez économisés et gagné au cours du dernier tour. Cependant, vous devrez également engager et dépenser vos fonds pour développer et protéger votre réseau commercial, en trouvant un équilibre entre les dividendes et la croissance.</p><p>Vous gagnerez des fonds au fur et à mesure d'autres factions achètent vos <strong>services</strong>, que ce soit l'utilisation de vos bateaux fluviaux ou de guerriers mercenaires, ou même l'une des cartes de votre main publique, qui est toutes <strong>à vendre</strong>. Vous pouvez définir les prix de vos services à chaque tour, alors essayez de prévoir de quels services les autres factions auront besoin.</p><p>Construire des comptoirs commerciaux vous aide à vous développer d votre clientèle et vendez plus de services, car une faction peut vous acheter plus de services à mesure qu'elle s'étend dans plus de clairières où vous avez des comptoirs commerciaux. Mais attention, lorsque vos comptoirs sont détruits, ils ne peuvent pas être reconstruits ! Bien que vous ne contrôliez pas beaucoup de clairières, vous pouvez toujours vous déplacer le long des chemins fluviaux puisque vous êtes une faction de <strong>nageurs</strong>.</p>",
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.